### PR TITLE
Add page for KNX fan component

### PR DESCRIPTION
**Pull request by itineric**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#16434

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/fan.knx.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+```yaml
+# Example configuration.yaml entry
+fan:
+  - platform: knx
+    name: living-room-fan
+    speed_address: '1/3/7'
+    speed_state_address: '1/3/8'
+```
+
+Configuration variables:
```
Comment (tjorim) [inline source/_components/fan.knx.markdown]: Replace this section according to the docs: https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration

**Review by tjorim — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
@itineric, @tjorim is correct. Could you please update the PR? 👍

File: source/_components/fan.knx.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
```
Comment (frenck) [inline source/_components/fan.knx.markdown]: You need to remove this header as well, this is added automatically by the configuration tags.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_components/fan.knx.markdown [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: knx.png
+ha_category: Fan
+ha_release: 0.78.0
+ha_iot_class: "Local Polling"
+---
+
```
Comment (tjorim) [inline source/_components/fan.knx.markdown:15]: Remove one of these empty lines

File: source/_components/fan.knx.markdown [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    speed_address: '1/3/7'
+    speed_state_address: '1/3/8'
+```
+
+{% configuration %}
+speed_address:
+  description: KNX group address for seeting the fan speed.
+  required: true
+  type: string
+name:
```
Comment (tjorim) [inline source/_components/fan.knx.markdown:39]: Name has a default value `default: KNX Fan`

File: source/_components/fan.knx.markdown [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+## {% linkable_title Configuration %}
+
+To use your KNX fan in your installation, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+fan:
+  - platform: knx
+    name: living-room-fan
```
Comment (tjorim) [inline source/_components/fan.knx.markdown:29]: The example may only contain the required variables to make it work, so remove `name` and `speed_state_address`

**Review by tjorim — CHANGES_REQUESTED** [review]
Some minor things @itineric, take a look at them.
Please fix the merge conflict as well!

⚠️ The `ha_release` will need an update before merging this one in.

**Pull request closed without merging**